mcs/class/System/ReferenceSources/SecureStringHelper.cs

```diff
@@ -0,0 +1,103 @@
+using System.Security;
+using System.Runtime.InteropServices;
+
+namespace System.Net
+{
+	static partial class UnsafeNclNativeMethods
+	{
+        [SuppressUnmanagedCodeSecurity]
+        internal unsafe static class SecureStringHelper
+        {
+#if DEBUG
+            // this method is only called as part of an assert
+            internal static bool AreEqualValues(SecureString secureString1, SecureString secureString2)
+            {
+                IntPtr bstr1 = IntPtr.Zero;
```

Comment:
That code isn't compiled at all, I just kept it to have a 1:1 copy of the code in referencesource.